### PR TITLE
Remove identity artist pick path

### PR DESCRIPTION
**Pull request by michellebrier**
### Description
Artist pick lives in discovery now; remove legacy identity read/writes and read/write directly to the artist_pick_track_id field in the user metadata in discovery.

### Dragons

*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

### How Has This Been Tested?
Tested reads and writes in web and mobile.

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### Feature Flags ###
Removes READ_ARTIST_PICK_FROM_DISCOVERY feature flag

### CODE REVIEW

File: packages/web/src/common/store/pages/profile/lineups/tracks/sagas.js [hunk 1]
```diff
@@ -39,80 +38,18 @@ function* getTracks({ offset, limit, payload, handle }) {
...
-  )
   const sort = payload?.sort === TracksSortMode.POPULAR ? 'plays' : 'date'
   const getUnlisted = true
-  const getFeatureEnabled = yield getContext('getFeatureEnabled')
-  const readArtistPickFromDiscoveryEnabled = yield call(
-    getFeatureEnabled,
-    FeatureFlags.READ_ARTIST_PICK_FROM_DISCOVERY
-  ) ?? false
 
-  if (!readArtistPickFromDiscoveryEnabled && user._artist_pick) {
```
Comment (michellebrier) [inline packages/web/src/common/store/pages/profile/lineups/tracks/sagas.js:60]: Lineup will already be sorted with artist pick first in the discovery API, which reads the `artist_pick_track_id` in the discovery users table

**Review by sliptype — APPROVED** [review]
This is awesome, soooo much cleaner!

Looks like there are a few lint errors, if you don't have it yet you should install the Prettier vscode extension and enable auto-format on save in the vscode settings

**Comment by michellebrier** [comment]
> This is awesome, soooo much cleaner!
> 
> Looks like there are a few lint errors, if you don't have it yet you should install the Prettier vscode extension and enable auto-format on save in the vscode settings

Thanks! Yeah I need to fix my lint setup... I use vim so its a little different

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/mbrier/plat-634/client-remove-artist-pick-legacy-code

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/mbrier/plat-634/client-remove-artist-pick-legacy-code

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/mbrier/plat-634/client-remove-artist-pick-legacy-code

**Pull request merged**